frontend/src/app/page.tsx

```
@@ -0,0 +1,303 @@
+        <div className="flex flex-1 justify-end gap-8">
+          <div className="hidden items-center gap-9 md:flex">
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="#features">기능</a>
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="#pricing">가격</a>
+            <a className="text-white/90 hover:text-white text-sm font-medium transition-colors" href="#">문의하기</a>
+          </div>
+          <div className="hidden md:flex items-center">
+            <NextLink href="/login" className="flex items-center justify-center rounded-lg bg-[linear-gradient(135deg,#6d28d9_0%,#8b5cf6_50%,#a78bfa_100%)] px-4 py-2 text-sm font-bold text-white transition-all hover:brightness-110 shadow-lg shadow-primary/20 border-t border-white/20">
+              Log In
+            </NextLink>
+          </div>
+          <button className="md:hidden text-white">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)